universal-application-tool-0.0.1/app/views/BaseHtmlView.java

```diff
@@ -33,6 +33,12 @@ protected Tag textField(String id, String fieldName, String labelText) {
         .attr("for", fieldName);
   }
 
+  protected Tag textFieldWithValue(String fieldName, String labelText, String placeholder) {
+    return label(
+            text(labelText), input().withType("text").withName(fieldName).withValue(placeholder))
```

Comment:
question: I saw somewhere in chat we're leaning towards the `.with()` syntax - is that possible for `label`s, too? Might make this a little more intuitive to read

Reply:
Let's do a follow-up PR to clean that up.